docs/content/core/crud.fsx

```diff
@@ -121,6 +121,27 @@ mvps2
 
 ctx.SubmitUpdates()
 
+(** update a single row 
+*)
+
+type Employee2 = {
```

Comment:
I added for the updateEmployee function because the Employee record above didn`t have an Id field